src/styleRe.re

```diff
@@ -15,9 +14,8 @@ type style =
       string [ | `value AnimatedRe.Value.t | `interpolation AnimatedRe.Interpolation.t];
 
 let combine a b => {
```

Comment:
Nevermind, it won't work if one combines the styles from stylesheet in a render.